msal/build.gradle

```diff
@@ -45,7 +45,7 @@ android {
 //        }
 //    }
 
-    /*
+
     //Commenting out until the next major version of common/msal/etc...
```

Comment:
remove the comment too?

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1747/commits/d8cd7660f27322126e297f10612573a567ef5292